src/components/Header/index.less

```diff
@@ -50,22 +44,82 @@
       .react-geo-nominatimsearch.ant-select-auto-complete {
         width: 60%;
       }
-
-      @media screen and (max-width: 680px) {
-        .react-geo-nominatimsearch.ant-select-auto-complete {
-          width: 90%;
-        }
-      }
     }
+
     &.right-items {
       flex: 1;
       justify-content: flex-end;
     }
+  }
+
+  @media (max-width:599px) {
+
+    .title {
+      display: none;
+    }
+
+    .item-container {
+      display: flex;
+
+      &.left-items {
+        width: 60px;
+        max-width: 60px;
+
+        .logo {
+          width: 55px;
+          height: auto;
+        }
+      }
+
+      &.center-items {
+        align-items: center;
+        justify-content: center;
+        width: 100%;
+
+        .ant-select.ant-select-auto-complete.react-geo-nominatimsearch.ant-select-single.ant-select-allow-clear.ant-select-show-search {
+          width: 250px;
+
+          .ant-select-selector {
+            width: 100%;
+          }
+        }
+      }
 
-    @media screen and (max-width: 680px) {
       &.right-items {
-        display: none;
+        width: 60px;
+        max-width: 60px;
+        align-items: center;
+        justify-content: center;
+
+        .username {
+          display: none;
+        }
       }
     }
   }
-}
+
+  @media (min-width:600px) {
+    .title {
+      display: none;
+    }
+
+    .item-container.left-items {
+      width: 100px;
+      max-width: 100px;
+    }
+
+    .item-container.center-items {
+      width: 100%;
+      justify-content: center;
+
+      .ant-select.ant-select-auto-complete.react-geo-nominatimsearch.ant-select-single.ant-select-allow-clear.ant-select-show-search {
+        width: fit-content;
+      }
+    }
+
+    .item-container.right-items {
+      width: 100px;
+      max-width: 100px;
+    }
+  }
+}
```

Comment:
missing newline at end of file. Do you have the `editorconfig` plugin installed?

Reply:
that should add these automatically, and also configure other editor settings for you (whitespace indentation, etc.)